nix/sources.json

```diff
@@ -1,187 +1,187 @@
-        "homepage": "",
-        "owner": "nmattia",
-        "repo": "napalm",
-        "rev": "4db4f253c78cfa8d8e8defb45dfe25e04409142a",
-        "sha256": "1rzj7a3r6wcsfjlzs5fj1min3rk8d8k9g9v00kmygdl1696mnkrb",
-        "type": "tarball",
-        "url": "https://github.com/nmattia/napalm/archive/4db4f253c78cfa8d8e8defb45dfe25e04409142a.tar.gz",
-        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
-    },
-    "replica-x86_64-darwin": {
-        "builtin": false,
-        "description": "The replica binary. It must be updated together with the canister-sandbox binary.",
-        "rev": "d004accc3904e24dddb13a11d93451523e1a8a5f",
-        "sha256": "000d30a2720cb87fdd1114b50cf60dcb2246102ce66a59a53e3a8694733f1710",
-        "type": "file",
-        "url": "https://download.dfinity.systems/blessed/ic/d004accc3904e24dddb13a11d93451523e1a8a5f/sdk-release/x86_64-darwin/replica.gz",
-        "url_template": "https://download.dfinity.systems/blessed/ic/<rev>/sdk-release/x86_64-darwin/replica.gz"
-    },
-    "replica-x86_64-linux": {
-        "builtin": false,
-        "description": "The replica binary. It must be updated together with the canister-sandbox binary.",
-        "rev": "d004accc3904e24dddb13a11d93451523e1a8a5f",
-        "sha256": "6e95796ebbeecf74916a56633fdceb56d9575b698fdf40984d35c674454b96a8",
-        "type": "file",
-        "url": "https://download.dfinity.systems/blessed/ic/d004accc3904e24dddb13a11d93451523e1a8a5f/sdk-release/x86_64-linux/replica.gz",
-        "url_template": "https://download.dfinity.systems/blessed/ic/<rev>/sdk-release/x86_64-linux/replica.gz"
-    },
-    "sandbox-launcher-x86_64-darwin": {
-        "builtin": false,
-        "description": "The sandbox_launcher binary. It must be updated together with the replica binary.",
-        "rev": "d004accc3904e24dddb13a11d93451523e1a8a5f",
-        "sha256": "e95d031cb18330846311d44b83c7c2f1f2e552eb00f9f12427b43da0685870af",
-        "type": "file",
-        "url": "https://download.dfinity.systems/blessed/ic/d004accc3904e24dddb13a11d93451523e1a8a5f/sdk-release/x86_64-darwin/sandbox_launcher.gz",
-        "url_template": "https://download.dfinity.systems/blessed/ic/<rev>/sdk-release/x86_64-darwin/sandbox_launcher.gz"
-    },
-    "sandbox-launcher-x86_64-linux": {
-        "builtin": false,
-        "description": "The sandbox_launcher binary. It must be updated together with the replica binary.",
-        "rev": "d004accc3904e24dddb13a11d93451523e1a8a5f",
-        "sha256": "074b171dca8f5a36a17511d59efe0478b130dbbd57f89d17d8757090a9b84809",
-        "type": "file",
-        "url": "https://download.dfinity.systems/blessed/ic/d004accc3904e24dddb13a11d93451523e1a8a5f/sdk-release/x86_64-linux/sandbox_launcher.gz",
-        "url_template": "https://download.dfinity.systems/blessed/ic/<rev>/sdk-release/x86_64-linux/sandbox_launcher.gz"
-    }
+        "advisory-db": {
```

Comment:
Would you please revert these whitespace changes?  Other times that we have run `niv update` this has not happened.  It obscures the differences and will happen in the reverse direction after the next `niv update`.

Reply:
Don't know how I messed that up. Fixed